source/accessible-media/production-audio-description.html.erb.md

```diff
@@ -135,58 +128,51 @@ screen readers can read aloud as the text is delivered. See [examples of
 TTS descriptions](http://ncamftp.wgbh.org/ibm/dvs/). The basic workflow
 for TTS audio descriptions generally follows this pattern:
 
-### Basic workflow for creating pre-produced TTS audio descriptions:
-
-{::nomarkdown}
-<%= sample_start %>
-{:/nomarkdown}
-
 1. **Watch the video**<br>
     Note opportunities for descriptive narration (i.e., pauses in
     narration or dialog into which descriptions can be inserted).
+
 2. **Write the description script**<br>
-    Using a **caption-editing tool**, write the description script that
+    Using a caption-editing tool, write the description script that
     will be read aloud by a screen reader during the time of playback.
+
 3. **Assign timecodes to each description**<br>
     Each timecode will represent when the description should be read by
     the user's screen reader.
+
 4. **Export the description file**<br>
     Use the caption format (e.g., WebVTT or TTML) that is supported by
     the target player. This file will be transmitted during playback and
     will be read aloud by the user's screen reader.
 
-{::nomarkdown}
-<%= sample_end %>
-{:/nomarkdown}
-
 Below is an image of a caption editor being used to timestamp an audio-description script.
 
 ![A caption editor showing a timestamped description script.](dx_script.png)
 
-<p>Using the <code>track</code> element and the <code><a href="https://html.spec.whatwg.org/multipage/embedded-content.html#text-track-kind">kind</a></code> attribute, the descriptions can be delivered at the time of playback and a screen reader will read them aloud. Below is a code sample:</p>
+Using the `<track>` element and the `kind` attribute, the descriptions can be delivered at the time of playback and a screen reader will read them aloud. Below is a code sample:
 
 {::nomarkdown}
 <%= code_start %>
 {:/nomarkdown}
 
 ~~~html
 <video controls>
-  <source src="myvideo.mp4" type="video/mp4" />
-  <track kind="descriptions" class="visually-hidden" src="my_tts_descriptions.vtt" srclang="en" label="Descriptions" />
+  <source src="myvideo.mp4" type="video/mp4">
+  <track kind="descriptions" class="visually-hidden" src="my_tts_descriptions.vtt" srclang="en" label="Descriptions">
 </video>
 ~~~
 
 {::nomarkdown}
 <%= code_end %>
 {:/nomarkdown}
 
-The `kind` attribute will cause the description file to be received invisibly (e.g., off-screen) so sighted users will not see it, but screen readers will be aware of it. Screen readers will then read the description text as it is delivered, synchronized at the time of playback. Read more about [techniques for delivering TTS descriptions](https://www.w3.org/WAI/GL/wiki/Using_the_track_element_to_provide_audio_descriptions). See [functioning examples of TTS descriptions](http://ncamftp.wgbh.org/ibm/dvs/#videos) using the `track` element along with Javascript to illustrate how screen readers will read off-screen descriptions aloud.
```

Comment:
Instead of linking to the wiki of the WCAG/AG WG, link to the technique directly H96 – This technique is not listed as relevant on the bottom of the page, BTW.